### ISSUE TITLE
Determine breed from image

### ISSUE THREAD

**Issue by jkwening** [michaelgsuttles/saveallthepets-analysis#5]
Sid: Integrate Microsoft breed information through API

Note: Jupyter Notebook that uses the Google API was put in the `test` directory

**Comment by jkwening** [comment]
We ran into an issue with Microsoft model so building our own using https://www.kaggle.com/c/dog-breed-identification/kernels dataset.

**Comment by jkwening** [comment]
Getting ~75% validation accuracy with simple ResNet50 model trained via transfer learning. #7 

Steps to improvements:

* EDA of images to determine optimal transformation configuration to improve generalization
* Experimenting with other Imagenet networks and/or additional nodes.
* Experimenting with ensemble model like XGBoost
I suspect will get most improvement from using an Imagenet network as feature extraction and XGBoost as fully connected layer.
